.github/workflows/ci.yml

```diff
@@ -0,0 +1,87 @@
+name: Nginx image creator
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  REGISTRY: ghcr.io
```

Comment:
Let's configure it also as `input` - https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs

Reply:
Moved to inputs in the build-angular.yml